### DISCUSSION TITLE
What is the timeline for hierarchical cohort?

### DISCUSSION THREAD

**Discussion by Zhenshan-Jin** [kubernetes-sigs/kueue#2697] [Q&A] [answered]
Hi community, thank you for maintaining and developing such a great product. 🥰

I have been hearing the [hierarchical cohort](https://github.com/kubernetes-sigs/kueue/tree/main/keps/79-hierarchical-cohorts#proposal) for a while including the [public presentation](https://youtu.be/6k_8Go3u8Qk?t=1713) and also the existing [PRs](https://github.com/kubernetes-sigs/kueue/pull/2693). Is there a specific timeline for the delivery of this feature? 

Thank you!

**Comment by tenzen-y** [comment, marked as answer]
Thank you for your interested in the Kueue and Hierarchical Cohort feature.
We will include the feature in the next minor version (v0.9.0).

**Reply by Zhenshan-Jin** [reply]
Thank you!
